### PR TITLE
Feature/dependency 08

### PR DESCRIPTION
**Pull request by pmosk**

### CODE REVIEW

**Comment by andreise** [comment]
Let's remove file .DS_Store

**Comment by andreise** [comment]
I guess .DS_Store should be added into global vscode gitignore on the local machine
Unfortunately standard global vscode gitignore does not contain ignore setting for .DS_Store:
https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore

File: src/dependency/Dependency/ServiceProviderExtensions/Extensions.GetOrThrow.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ partial class ServiceProviderExtensions
                 return (T)service;
             }
 
-            throw new InvalidOperationException($"Service {typeof(T)} can not be resolved by service provider.");
+            throw new InvalidOperationException($"A service of the type {typeof(T)} can not be resolved by the service provider.");
```
Comment (andreise) [inline src/dependency/Dependency/ServiceProviderExtensions/Extensions.GetOrThrow.cs]: It seems '`the type {typeof(T)}`' is redundant and '`type {typeof(T)}`' is enough

Comment (pmosk) [reply src/dependency/Dependency/ServiceProviderExtensions/Extensions.GetOrThrow.cs]: Have done

**Pull request merged**